<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab01_LogCollection">
       <attribute name="title">
        <string>로그 폴더와 파일 모음</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_path">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="combox_extensionFiles"/>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_path">
          <property name="text">
           <string>확인</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_logAnalysis">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QTableWidget" name="tableWidget_directoryFilesView"/>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QProgressBar" name="progressBar_loading">
          <property name="value">
           <number>24</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_path">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab02_LogAnalysis">
       <attribute name="title">
        <string>로그 분석</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="8,1,1" columnstretch="5,5" rowminimumheight="8,1,1" columnminimumwidth="5,5">
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="listWidget_fileContentOutput"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="comboBox_currentFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_delete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
